docs/IE_DG/Samples_Overview.md

```diff
@@ -35,6 +35,8 @@ Inference Engine sample applications include the following:
    - [Object Detection for SSD C Sample](../../inference-engine/ie_bridges/c/samples/object_detection_sample_ssd/README.md)
    - [Object Detection for SSD Python* Sample](../../inference-engine/ie_bridges/python/sample/object_detection_sample_ssd/README.md)
  
+> **NOTE**: Samples support input paths containing only ASCII characters.
```

Comment:
This is not valid for hello classification sample which support UNICODE paths as well, right?

Reply:
Yes, it is right. But we decided to do not mentioned about it in this place. When user look at readme for hello classification sample, he will see that UNICODE path  supported in this sample.